### ISSUE TITLE
[bug] v5.1.2 doesn't trim spaces betweeen "url(" and ")"

### ISSUE THREAD

**Issue by gdh1995** [jakubpawlowicz/clean-css#1161]
Recently I tried updating clean-css in my project from v4.2.3 to latest. But v5 generated a bigger result file, and I found it doesn't trim spaces after `url(` and before a URL string.

### Environment

* clean-css version - `npm ls clean-css`: 5.1.2
* node.js version - `node -v`: v14.8.0
* operating system: Windows 10 x64

### Configuration options

```js
var CleanCSS = require('clean-css');
var input = `
a {
  background: url(
    "about:blank"
  );
}
`;
var options = { /* options */ };
var output = new CleanCSS(options).minify(input);
console.log(output.styles)
```

### Actual output CSS

```css
a{background:url(    "about:blank"  )}
```

### Expected output CSS

```css
a{background:url("about:blank")}
```

**Comment by jakubpawlowicz** [comment]
Thanks @gdh1995  for investigating the reason of larger builds and narrowing down the case. Let me verify it and issue a fix.

**Comment by jakubpawlowicz** [comment]
Fixed in 5.1.3.
